api/resource_releases.go

```diff
@@ -40,35 +40,33 @@ func (c *Client) GetAppReleasesNomad(ctx context.Context, appName string, limit
 	return data.App.Releases.Nodes, nil
 }
 
-func (c *Client) GetAppReleasesMachines(ctx context.Context, appName string, limit int) ([]Release, error) {
+func (c *Client) GetAppReleasesMachines(ctx context.Context, appName, status string, limit int) ([]Release, error) {
 	query := `
-		query ($appName: String!, $limit: Int!) {
-			app(name: $appName) {
-				releases: releasesUnprocessed(first: $limit) {
-					nodes {
-						id
-						version
-						description
-						reason
-						status
-						imageRef
-						stable
-						user {
```

Comment:
removing user here will break `fly apps releases` output

Reply:
Haaa, never meant to make that change, I'm pushing a fix